html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,74 @@
 <?php
 $limit = $_GET['target'];
 
-$dsn = 'mysql:dbname=test;host=mysql';
-$dbuser = 'test';
-$dbpassword = 'test';
+
+if($limit <= 0
+    or $limit != (int)$limit
+    or $limit != is_numeric($limit)
+    or strpos($limit,'.0')){
+
+    http_response_code(500);
+    exit();
+
+}else{
+    $dsn = 'mysql:dbname=test;host=mysql';
+    $dbuser = 'test';
+    $dbpassword = 'test';
+    try{
+        $db = new PDO($dsn,$dbuser,$dbpassword);
+    }catch (PDOException $e){
+        echo 'DB接続エラー:'.$e->getMessage();
+        http_response_code(404);
+        exit();
+    }
+    $records = $db->query('SELECT value FROM prechallenge3');
+    $record = $records->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
(修正不要)
何が入っているか想像しやすい変数名にできるとより良いコードになります！
```suggestion
    $pdo = $db->query('SELECT value FROM prechallenge3');
    $records = $records->fetchAll(PDO::FETCH_ASSOC);
```